tests/test_bench.py

```
@@ -0,0 +1,23 @@
+    common.start_redis(release=True)
+
+
+def teardown_module(module):
+    common.shutdown_redis()
+
+
+def setup_function(function):
+    if common.g.name in common.client.list_graphs():
+        common.g.delete()
+
+def query(query: str, params=None):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)